src/js/control-bar/time-controls/time-display.js

```diff
@@ -77,12 +88,19 @@ class TimeDisplay extends Component {
       return;
     }
 
-    while (this.contentEl_.firstChild) {
-      this.contentEl_.removeChild(this.contentEl_.firstChild);
-    }
+    const oldNode = this.textNode_;
 
     this.textNode_ = document.createTextNode(this.formattedTime_ || this.formatTime_(0));
-    this.contentEl_.appendChild(this.textNode_);
+
+    if (!this.textNode_) {
+      return;
+    }
+
+    if (oldNode) {
+      this.contentEl_.replaceChild(this.textNode_, oldNode);
```

Comment:
replaceChild appears to be about 10% faster than appendChild, and we call this function a lot